[docs] Describe requestOptions.isAjax (#5698)
DevExpress/testcafe: docs/articles/templates/intercept-http-requests/requestoptions-object.md

The `RequestOptions` object contains the request parameters.

Property      | Type    | Description
------------- | ------- | ------------
`headers`     | Object  | The request headers in the property-value form.
`body`        | [Buffer](https://nodejs.org/api/buffer.html) | The request body.

> `url`         | String  | The URL to which to send the request.

Reviewer: that's a mouthful. Would this work:
```suggestion
`url`         | String  | The URL of the resource.
```

Author: Shorter and works. Applied

> `protocol`    | String  | The protocol to use. Default: *http:*.

Reviewer: Why not:
```suggestion
`protocol`    | String  | The request protocol. Default: *http:*.
```

Author: Seems same to me

Reviewer 2: I agree with this correction.

`hostname`    | String  | The alias for the host.

> `host`        | String  | The domain name or IP address of the server to issue the request to. Default: *localhost*.

Reviewer: Honest question: what's the difference b/w `host` and `hostname`? I don't quite understand

Author: No clue
@miherlosev ?

Reviewer 3: https://nodejs.org/api/url.html#url_url_host

Reviewer: The difference is not obvious from the documentation. Here's what I suggest:

```suggestion
`hostname`    | String  | The domain name or IP address of the destination server. Default: *localhost*.
`host`        | String  | The domain name or IP address of the destination server, including the listening port. Default: *localhost:80*.
```

Reviewer: @DIRECTcut @miherlosev

`port`        | Number  | The port of the remote server. Default: *80*.

> `path`        | String  | The request path. Should include query string if any. For example, *'/index.html?page=12'*. An exception is thrown when the request path contains illegal characters (for example, spaces). Default: *'/'*.

Reviewer 2: ```suggestion
`path`        | String  | The request path. Should include the query string if any. For example, *'/index.html?page=12'*. An exception is thrown when the request path contains illegal characters (e.g.,, spaces). Default: *'/'*.
```

Author: applied

> `method`      | String  | The string specifying the HTTP request method. Default: *'GET'*.

Reviewer: ```suggestion
`method`      | String  | The HTTP request method. Default: *'GET'*.
```

`credentials` | Object  | Credentials that were used for authentication in the current session with NTLM or Basic authentication. For HTTP Basic authentication, these are `username` and `password`. NTLM authentication additionally specifies `workstation` and `domain`. See [HTTP Authentication](/testcafe/documentation/guides/advanced-guides/authentication.html#http-authentication).
`proxy`       | Object  | If a proxy is used, the property contains information about its `host`, `hostname`, `port`, `proxyAuth`, `authHeader` and `bypassRules`.

> `isAjax`      | Boolean | Specifies whether the request is AJAX.

Reviewer 3: `Specifies whether the request is AJAX request (xhr or fetch).`

Author: Edited
